### DISCUSSION TITLE
how can i debug c++?

### DISCUSSION THREAD

**Discussion by loldruger** [helix-editor/helix#5631] [Q&A]
![image](https://user-images.githubusercontent.com/9137739/213909033-758b407c-9051-4723-9b39-1d7cdc30677f.png)
its all ok

but when i type :debug-start or :debug-remote, 
![image](https://user-images.githubusercontent.com/9137739/213909071-1447054f-5f4b-469b-bf2c-83541bb266fb.png)

this happens.. what can i do for it? im on freebsd

**Comment by helmesjo** [comment]
For me this usually happens when the path to the executable I typed in was wrong (you can type in some garbage and you'll see this error).
